nextcord/client.py

```diff
@@ -193,6 +196,20 @@ class Client:
 
         .. versionadded:: 2.0
 
+    # TODO: Give better description
```

Comment:
this will show in the docs wont it? aybe put the todo after the closing ``"""`` or something